src/main/java/cms/gov/madie/measure/services/MeasureLockService.java

```diff
@@ -3,7 +3,7 @@
 import cms.gov.madie.measure.dto.LockInfo;
 import cms.gov.madie.measure.locks.MeasureLock;
 import cms.gov.madie.measure.repositories.MeasureLockRepository;
-import cms.gov.madie.measure.resources.DuplicateKeyException;
+import org.springframework.dao.DuplicateKeyException;
```

Comment:
If changing from our exception to the spring's was intentional, can our exception class be deleted?